From e54f632085f5dde039defc7e77e91b10fb86f3d5 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Fri, 31 Aug 2012 14:59:25 +0100
Subject: [PATCH 28/29] EPEL 5: Disable tar --xz test.

Although xz is available, ancient tar didn't have the --xz option.
---
 generator/generator_actions.ml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/generator/generator_actions.ml b/generator/generator_actions.ml
index 79ed8f4..89c553f 100644
--- a/generator/generator_actions.ml
+++ b/generator/generator_actions.ml
@@ -3655,7 +3655,7 @@ To get the checksums for many files, use C<guestfs_checksums_out>." };
         [["mkdir"; "/tar_in_gz"];
          ["tar_in"; "../data/helloworld.tar.gz"; "/tar_in_gz"; "gzip"];
          ["cat"; "/tar_in_gz/hello"]], "hello\n");
-      InitScratchFS, IfAvailable "xz", TestOutput (
+      InitScratchFS, Disabled, TestOutput (
         [["mkdir"; "/tar_in_xz"];
          ["tar_in"; "../data/helloworld.tar.xz"; "/tar_in_xz"; "xz"];
          ["cat"; "/tar_in_xz/hello"]], "hello\n")
-- 
1.7.4.1

